frontend/.eslintrc

```diff
@@ -55,7 +57,8 @@
     "plugins": [
         "babel",
         "react",
-        "prettier"
+        "prettier",
+        "simple-import-sort"
```

Comment:
It turns out the built in sort-imports rule [won't fix anything automatically](https://github.com/eslint/eslint/issues/11542), so a plugin is required

Reply:
very cool!